test/e2e/nsx_security_policy_test.go

```diff
@@ -35,4 +38,20 @@ func TestSecurityPolicyBasicTraffic(t *testing.T) {
 
 	err = testData.runPingCommandFromTestPod(ns, "busybox", iPs, 4)
 	assert.NotNilf(t, err, "Error when running ping command from test Pod 'busybox'")
+
+	tagScopeClusterKey := strings.Replace(common.TagScopeNamespace, "/", "\\/", -1)
```

Comment:
Perhaps it's better to encapsulate those lines into a function, e.g. 

testData.assertSecurityPolicyStatus(ns, spName, exist)
testData.assertRuleStatus(ns, ruleName, notExist)

Then other test cases could simply use these two functions to assert status before creating securitypolicy and after deleting it.

Reply:
it's out the scope of this patch